### DISCUSSION TITLE
OK so...

### DISCUSSION THREAD

**Discussion by asim** [micro/go-micro#2233] [General]
OK so I mostly put this project on the back burner to focus on m3o.com and turn micro into more of a platform, but it looks like people still want to use Go Micro. For whatever reason, it's still a thing of value. I'm not personally dealing with issues, instead I'm accepting PR's and creating issues for potential feature work where it makes sense. 

Everyone is free to contribute and let's see where this thing goes in the hands of the community.

Peace
Asim

**Comment by unvoodoo** [comment]
Both micro and go-micro rock, we started to test/use them recently and we enjoy the style and concepts of these projects, thanks man.  
Started with micro first, but then decided to switch to go-micro.

**Reply by Davincible** [reply]
@unVooDoo Could you please elaborate on the switch? I am a bit confused now as to the state of both projects. 

It seemed that go-micro was replaced in favour of micro, but now go-micro has been getting a lot of attention the last few months? So I'm not sure what to use anymore

**Reply by asim** [reply]
They serve different purposes. Micro is an all encompassing platform that powers M3O. Where as Go Micro is a pluggable framework that only focuses on developing Go services. A lot of people choose Go Micro because they want to build the rest of the stack using other open source tools.

**Reply by Davincible** [reply]
I see, thank you for your reply :) 
And which of the two projects is currently most actively developed and maintained? 
Looking at both commit histories I see you made commits to both, so I wasn't sure.

For me I am just trying to decide which project to use to develop my microservices, both projects sound nice, and now that go-micro also has the CLI they have come really close from where I am standing

**Reply by asim** [reply]
Most developers prefer Go Micro. It's maintained by the community. Micro is used and maintained by my company M3O.

**Reply by Davincible** [reply]
Still thanks for your explanation. Using go-micro now and after some struggle to fully grasp the ideas behind the framework it now works wonderfully
